wasm/legend/legendTxTypes/constants.go

```diff
@@ -39,6 +39,7 @@ const (
 	NilNonce          = 0
 	NilExpiredAt      = 0
 	NilTxAccountIndex = int64(-1)
+	NilTxAssetId      = int64(-1)
```

Comment:
Can we rename these constants to keep the same with zkbnb repo?
![image](https://user-images.githubusercontent.com/90749943/189270530-a409ef57-7a7b-4e59-b71d-eceebe8e0275.png)

Reply:
let check